Src/PCompiler/CompilerCore/Backend/Java/MachineCodeGenerator.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Plang.Compiler.Backend.Java {
 
-    internal class JavaCodeGenerator : ICodeGenerator
```

Comment:
Let this be JavaCodeGenerator as it contains everything else .. types, global functions, machines, monitors, UD types ...

Reply:
OK, I'll rename this to be something more specific once the patch for types is out too.